packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -59,6 +61,45 @@ export interface LaunchRequestArguments
   trace?: boolean | string;
 }
 
+export class ApexVariable extends Variable {
+  public type: string;
+  public constructor(name: string, value: string, type: string) {
+    super(name, value);
+    this.type = type;
+  }
+}
+
+export class ApexDebugStackFrameInfo {
+  public readonly frameNumber: number;
+  public readonly signature: string;
+  public globals: Map<String, Variable>;
+  public statics: Map<String, Variable>;
+  public locals: Map<String, Variable>;
+  public constructor(frameNumber: number, signature: string) {
+    this.frameNumber = frameNumber;
+    this.signature = signature;
+    this.globals = new Map<String, Variable>();
+    this.statics = new Map<String, Variable>();
+    this.locals = new Map<String, Variable>();
+  }
+}
+
+export enum SCOPE_TYPES {
+  LOCAL = 'local',
+  STATIC = 'static',
+  GLOBAL = 'global'
+}
+
+export class ScopeContainer {
```

Comment:
This class is not necessarily needed right now but will be when I implement objects/collections. Wanted to just keep it so it wouldn't throw me off.